code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -29,48 +43,51 @@
 	speak_emote = list("chitters")
 	emote_hear = list("chitters")
 	speak_chance = 5
+	speed = 0
 	turns_per_move = 5
 	see_in_dark = 4
 	butcher_results = list(/obj/item/food/meat/slab/spider = 2, /obj/item/food/spiderleg = 8)
 	response_help_continuous = "pets"
 	response_help_simple = "pet"
 	response_disarm_continuous = "gently pushes aside"
 	response_disarm_simple = "gently push aside"
-	maxHealth = 200
-	health = 200
-	obj_damage = 60
-	melee_damage_lower = 15
-	melee_damage_upper = 20
+	initial_language_holder = /datum/language_holder/spider
+	maxHealth = 80
+	health = 80
+	damage_coeff = list(BRUTE = 1, BURN = 1, TOX = 1, CLONE = 1, STAMINA = 1, OXY = 1)
+	obj_damage = 30
+	melee_damage_lower = 20
+	melee_damage_upper = 25
 	faction = list("spiders")
-	var/busy = SPIDER_IDLE
 	pass_flags = PASSTABLE
 	move_to_delay = 6
-	ventcrawler = VENTCRAWLER_ALWAYS
 	attack_verb_continuous = "bites"
 	attack_verb_simple = "bite"
 	attack_sound = 'sound/weapons/bite.ogg'
 	unique_name = 1
 	gold_core_spawnable = HOSTILE_SPAWN
 	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE
 	footstep_type = FOOTSTEP_MOB_CLAW
-	var/playable_spider = FALSE
+	poison_per_bite = 0
+	///Whether or not the spider is in the middle of an action.
+	var/is_busy = FALSE
+	///How quickly the spider can place down webbing.  One is base speed, larger numbers are slower.
+	var/web_speed = 1
+	///The web laying ability
 	var/datum/action/innate/spider/lay_web/lay_web
-	var/directive = "" //Message passed down to children, to relay the creator's orders
+	///The message that the mother spider left for this spider when the egg was layed.
+	var/directive = ""
 
 /mob/living/simple_animal/hostile/poison/giant_spider/Initialize()
 	. = ..()
 	lay_web = new
 	lay_web.Grant(src)
 
-/mob/living/simple_animal/hostile/poison/giant_spider/Destroy()
-	QDEL_NULL(lay_web)
-	return ..()
-
-/mob/living/simple_animal/hostile/poison/giant_spider/Topic(href, href_list)
-	if(href_list["activate"])
-		var/mob/dead/observer/ghost = usr
-		if(istype(ghost) && playable_spider && !(GLOB.ghost_role_flags & GHOSTROLE_SPAWNER))
-			humanize_spider(ghost)
+/mob/living/simple_animal/hostile/poison/giant_spider/Life(mapload)
+	. = ..()
+	staminaloss = max(0, staminaloss - 10)
```

Comment:
Really not a fan of implementing stamina processing as a spider snowflake here.

Reply:
What are your thoughts on it, then?  I'm not keen on adding stamina to all simple mobs, as that'd be something for a different PR I'd say.